src/murmur/murmur_pch.h

```diff
@@ -98,5 +89,15 @@ extern "C" {
 #include <dns_sd.h>
 #endif
 
+#include <openssl/aes.h>
```

Comment:
What was the reason for moving this include block down? Did including the ssl.h create issues? If so a note somewhere would be nice.